test/adapters/jasmine.html

```diff
@@ -2,23 +2,37 @@
 <html>
 	<head>
 		<title>Jasmine Spec Runner</title>
-		<link rel="shortcut icon" type="image/png" href="../../lib/jasmine/images/jasmine_favicon.png">
-		<link rel="stylesheet" type="text/css" href="../../lib/jasmine/lib/jasmine-core/jasmine.css">
+		<link rel="shortcut icon" type="image/png" href="../../node_modules/jasmine/node_modules/jasmine-core/images/jasmine_favicon.png">
+		<link rel="stylesheet" type="text/css" href="../../node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/jasmine.css">
 	</head>
 
 	<body>
-		<script src="../../lib/jasmine/lib/jasmine-core/jasmine.js"></script>
-		<script src="../../lib/jasmine/lib/jasmine-core/jasmine-html.js"></script>
+		<script src="../../node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/jasmine.js"></script>
+		<script src="../../node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/jasmine-html.js"></script>
+		<script src="../../node_modules/jasmine/node_modules/jasmine-core/lib/jasmine-core/boot.js"></script>
```

Comment:
This stuff is ugly, but I wasn't concerned with doing this the best way.  It used to grab these files from Bower.  This should probably be updated to be more correct.